tutorial/pyam_first_steps.ipynb

```diff
@@ -182,6 +182,31 @@
     "df.filter(model='MESSAGE*')[['model', 'scenario']].drop_duplicates()"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Using keyword `keep=False` allows you to discard everything that is found by the filter rather than keeping it"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "df.filter(region=\"World\", keep=False).data[['region']].drop_duplicates()"
```

Comment:
There is a function `df.regions()` which does the same thing as `df.data[['region']].drop_duplicates()`

Reply:
oops good pickup